core/state_processor.go

```diff
@@ -1859,6 +1925,14 @@ func ProcessQiTx(tx *types.Transaction, chain ChainContext, checkSig bool, isFir
 
 			utxosCreatedDeleted.UtxosCreatedHashes = append(utxosCreatedDeleted.UtxosCreatedHashes, types.UTXOHash(tx.Hash(), uint16(txOutIdx), utxo))
 			utxosCreatedDeleted.UtxosCreatedKeys = append(utxosCreatedDeleted.UtxosCreatedKeys, rawdb.UtxoKeyWithDenomination(tx.Hash(), uint16(txOutIdx), utxo.Denomination))
+			if indexAddressUtxos {
```

Comment:
dont we need this in the trimBlock?

Reply:
We don't create UTXOs in trimBlock we delete them, and the deleted ones are returned from finalize